siliconcompiler/tools/yosys/sc_syn.tcl

```diff
@@ -61,7 +61,7 @@ if { [file exists "inputs/$sc_design.v"] } {
 yosys chparam -list
 if {[dict exists $sc_cfg option param]} {
     dict for {key value} [dict get $sc_cfg option param] {
-        if !{[string is integer $value]} {
+        if {![string is integer $value]} {
```

Comment:
This also works as intended in this case, but in the general case I think `!{[cmd...]}` can behave funny